models/balancer/ethereum/balancer_ethereum_pools_tokens_weights.sql

```diff
@@ -0,0 +1,129 @@
+{{
```

Comment:
I'd instead create 2 separate spells, one for V1 and another for V2. Then, this query would be basically a `UNION` of those 2 spells.

Reply:
This is now done.